FAQ.md

```diff
@@ -246,6 +247,36 @@ To run and train InstructLab locally, you must meet the following requirements:
 - Approximately 60GB of free disk space is needed to run the entire process locally on Apple hardware
 - About 32 GB RAM
 
+## Glossary
+
+| Term | Explanation | Additional Reference |
+| --- | --- | --- |
+| Checkpoints | Snapshots during training. They are scored individually and the best is selected. | N/A |
+| CUDA | “Compute Unified Device Architecture” - A parallel computing platform and API for general computing on GPUs by NVIDIA. | [Ref](https://www.nvidia.com/en-us/geforce/technologies/cuda/) |
+| DeepSpeed | Deep learning optimization library for PyTorch | [Ref](https://www.deepspeed.ai/) |
+| Granite | Open source LLM released by IBM | [Ref](https://huggingface.co/ibm-granite) |
```

Comment:
```suggestion
| Granite | Open source licensed LLM released by IBM | [Ref](https://huggingface.co/ibm-granite) |
```

Reply:
https://github.com/instructlab/community/pull/288#discussion_r1630129485

Reply:
I'm fine to add it back in! cc @markstur